Sources/MapboxNavigation/PassiveLocationProvider.swift

```diff
@@ -10,7 +10,7 @@ import MapboxMaps
  
  This class depends on `PassiveLocationDataSource` to detect the user’s location as it changes. If you want location updates but do not need to display them on a map and do not want a dependency on the MapboxNavigation module, you can use `PassiveLocationDataSource` instead of this class.
```

Comment:
Now that PassiveLocationManager has been renamed PassiveLocationProvider, we can rename PassiveLocationDataSource to PassiveLocationManager for consistency with NavigationLocationManager. (Or, if you prefer, we could rename both classes to end with “LocationDataSource”.)

Reply:
Thanks for the suggestion, I'll rename the `PassiveLocationDataSource` to `PassiveLocationManager` for consistency